### DISCUSSION TITLE
Failed to pathch lockfile,please try uninstalling and reinstalling next in this workspace #71176

### DISCUSSION THREAD

**Discussion by hexiaochen666** [vercel/next.js#71177] [Help] [answered]
### Summary

Failed to pathch lockfile,please try uninstalling and reinstalling next in this workspace #71176  nextjs 12.1.6
![image](https://github.com/user-attachments/assets/d1ccb594-bb98-46f1-8c10-e43e35accbd5)

**Comment by SumitKumar-17** [comment]
Remove the node_modules folder and the package-lock.json  or yarn.lock file present in your project.
After that use npm install or yarn install.

**Reply by joecotton-stg** [reply]
This worked for me

**Comment by hexiaochen666** [comment, marked as answer]
add .babelrc https://nextjs.org/docs/messages/failed-loading-swc
